test/cjs/default-require.js

```diff
@@ -0,0 +1,326 @@
+/* eslint-disable no-empty */
+/* eslint-disable no-unused-vars */
+/**
+ * Mnemonist CommonJS Default Require Testing
+ * ===========================================
+ */
+const BiMap = require('mnemonist/bi-map');
+const BitSet = require('mnemonist/bit-set');
+const BitVector = require('mnemonist/bit-vector');
+const BKTree = require('mnemonist/bk-tree');
+const BloomFilter = require('mnemonist/bloom-filter');
+const CircularBuffer = require('mnemonist/circular-buffer');
+const DefaultMap = require('mnemonist/default-map');
+const FibonacciHeap = require('mnemonist/fibonacci-heap');
+const {MinFibonacciHeap} = require('mnemonist/fibonacci-heap');
+const FixedReverseHeap = require('mnemonist/fixed-reverse-heap');
+const FixedStack = require('mnemonist/fixed-stack');
+const FuzzyMap = require('mnemonist/fuzzy-map');
+const FuzzyMultiMap = require('mnemonist/fuzzy-multi-map');
+const HashedArrayTree = require('mnemonist/hashed-array-tree');
+const Heap = require('mnemonist/heap');
+const {MaxHeap} = require('mnemonist/heap');
+const InvertedIndex = require('mnemonist/inverted-index');
+const LinkedList = require('mnemonist/linked-list');
+const LRUCache = require('mnemonist/lru-cache');
+const LRUCacheWithDelete = require('mnemonist/lru-cache-with-delete');
+const LRUMap = require('mnemonist/lru-map');
+const LRUMapWithDelete = require('mnemonist/lru-map-with-delete');
+const MultiMap = require('mnemonist/multi-map');
+const MultiSet = require('mnemonist/multi-set');
+const PassjoinIndex = require('mnemonist/passjoin-index');
+const Queue = require('mnemonist/queue');
+const set = require('mnemonist/set');
+const SparseSet = require('mnemonist/sparse-set');
+const Stack = require('mnemonist/stack');
+const StaticDisjointSet = require('mnemonist/static-disjoint-set');
+const StaticIntervalTree = require('mnemonist/static-interval-tree');
+const {GeneralizedSuffixArray} = require('mnemonist/suffix-array');
+const SymSpell = require('mnemonist/symspell');
+const Trie = require('mnemonist/trie');
+const TrieMap = require('mnemonist/trie-map');
+const Vector = require('mnemonist/vector');
+const {Uint16Vector} = require('mnemonist/vector');
+const VPTree = require('mnemonist/vp-tree');
+
```

Comment:
Is checking the fact we can correctly require stuff not enough here? I am asking because it looks like there is a lot of duplication happening here and it could be tedious to maintain.

Reply:
Unfortunately, no => successful import but object not usable can happen if the export is not properly done (wrong object target like for `Float64Vector` issue, default used instead of named, ...)
And I do not see how to test it without duplicating the code, at least for now.

Reply:
I have an idea to consolidate named exports testing for CommonJS, ESM and TS: use the existing TS file for types test to generate CommonJS, ESM files into the fake consuming apps and then run the tests. 

Direct default exports will still have to stay as it is.

Reply:
But that can be done later on, I will create an issue to track the cleanup.